lib/config/migrations/custom/rebase-stale-prs-migration.ts

```diff
@@ -0,0 +1,24 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class RebaseStalePrsMigration extends AbstractMigration {
+  override readonly deprecated = true;
+  readonly propertyName = 'rebaseStalePrs';
+
+  override run(value): void {
+    const rebaseConflictedPrs = this.get('rebaseConflictedPrs');
+
+    if (rebaseConflictedPrs !== false) {
```

Comment:
IMO the double negative is a little confusing

```suggestion
    if (rebaseConflictedPrs === true) {
```

or 

```suggestion
    if (rebaseConflictedPrs) {
```

Reply:
Now I re-read this, I realise that `rebaseConflictedPrs` is `boolean | undefined` so you'd need `rebaseConflictedPrs === true || rebaseConflictedPrs === undefined`. Gotta love that trinary logic 😅